main.tf

```diff
@@ -92,9 +92,37 @@ resource "aws_lambda_function" "ses_redirecter" {
   }
 }
 
+data "aws_route53_zone" "selected" {
+  name  = "${var.domain}."
+  count = 1
+}
+
+resource "aws_ses_domain_identity" "domain" {
+  domain = var.ses_domain
+}
+
+resource "aws_route53_record" "mx" {
+  zone_id = data.aws_route53_zone.selected[0].zone_id
+  name    = var.ses_domain
+  type    = "MX"
+  ttl     = "600"
+  records = ["10 inbound-smtp.${var.aws_region}.amazonaws.com"]
+
+  count = 1
+}
+
+resource "aws_route53_record" "verification" {
+  zone_id = data.aws_route53_zone.selected[0].zone_id
+  name    = "_amazonses.${var.ses_domain}"
+  type    = "TXT"
+  ttl     = "600"
+  records = [aws_ses_domain_identity.domain.verification_token]
+
+  count = 1
+}
 
 data "template_file" "instructions" {
-  template = file("${path.module}/instructions.txt")
+  template = file("${path.module}/instructions.md")
 
   vars = {
     aws_region = var.aws_region
```

Comment:
`aws_region` isn't in the template file. Do you want to keep it here?

Reply:
hmm, it's nice to support Route53 if someone else needs that, but we don't use Route53 so this wouldn't work for us. We need to use Cloudflare. So either we add support to use either depending on parameters sent to it, or maybe have a base model and one or two extended modules that make use of the base but then create dns records in the appropriate place

Reply:
Oh.  When I was doing it manually, I did it through Route53.